src/documents/ComponentGrades.md

```diff
@@ -14,121 +14,87 @@ The Infusion Framework already contains several predefined component grades that
 fluid.defaults("cspace.messageBarImpl", {
-    gradeNames: ["fluid.rendererComponent", "autoInit"],
+    gradeNames: ["fluid.rendererComponent"],
     ...
 });
 ```
 
 ```javascript
 fluid.defaults("cspace.util.relationResolver", {
-    gradeNames: ["fluid.modelComponent", "autoInit"],
+    gradeNames: ["fluid.modelComponent"],
     ...
 });
 ```
 
-## Initializing Graded Components ##
+## Initializing Components ##
 
-The Framework offers support for automatic initialization of graded component through `autoInit`. When the `autoInit` flag is added to the `gradeNames` array, the Framework will create the creator function automatically – the developer does not need to write a creator function.
+The framework will automatically construct a creator function for any component which is derived (even indirectly) from `fluid.component`:
 
-To use the `autoInit` flag, add it to the array of `gradeNames`, as shown below:
 
 ```javascript
 fluid.defaults("fluid.uploader.fileQueueView", {
-    gradeNames: ["fluid.viewComponent", "autoInit"],
+    gradeNames: ["fluid.viewComponent"],
     ...
 });
 
-
-var that = fluid.uploader.fileQueueView( ... ); // The framework has automatically generated this function since the component is autoInit
+// The framework has automatically generated this function since the grade is a component grade
+var that = fluid.uploader.fileQueueView( ... );
 ```
 
-<div class="infusion-docs-note"><strong>Note:</strong> The <code>autoInit</code> flag should always be used if you expect the grade to be directly instantiated as a component. It can be omitted if the only use of the grade is as an "add-on" ("<a href="https://en.wikipedia.org/wiki/Mixin">mixin</a>") to another grade hierarchy.</div>
-
 ## Combining Grades ##
 
-Since the `fluid.defaults` directive introduces a grade into the system, various components can be composed to create new ones. Options, fields and methods introduced by the ancestor grades will be merged. The merging happens, firstly in hierarchical order (grades comprising the ancestor grade are resolved before the actual component grades resolution) and secondly in the left-to-right order (defaults from the grade on the right taking precedence over the defaults from the grade on the left, more details can be found at the JIRA [FLUID-5085](http://issues.fluidproject.org/browse/FLUID-5085)). For example:
+Since the `fluid.defaults` directive introduces a grade into the system, various components can be composed to create new ones. Options, fields and methods introduced by the ancestor grades will be merged. 
+The merging happens, firstly in hierarchical order (grades comprising the ancestor grade are resolved before the actual component grades resolution) and secondly in the right-to-left order (defaults from the grade on the left taking precedence over the defaults from the grade on the right, more details can be found at the JIRA [FLUID-5085](http://issues.fluidproject.org/browse/FLUID-5085)). For example:
```

Comment:
Rather than pointing the reader to a rather context-specific JIRA ticket, can we just describe those "more details" here or elsewhere in the documentation?